packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -94,6 +94,26 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
    */
   staysActiveInBackground: boolean;
 
+  /**
+   * Float value indicating how far the player has buffered the video in seconds.
+   *
+   * This value is 0 when the player has not buffered up to the current playback time.
+   * When it's impossible to determine the buffer state (e.g. when the player isn't playing any media), this value is -1.
```

Comment:
```suggestion
   * When it's impossible to determine the buffer state (for example, when the player isn't playing any media), this value is -1.
```

Let's avoid using Latin abbreviations as per our writing style guide.